<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QWidget" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="home_button">
       <property name="text">
        <string>Home</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addBook_button">
       <property name="text">
        <string>Add book</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="allBooks_button">
       <property name="text">
        <string>All books</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addAuthor_button">
       <property name="text">
        <string>Add author</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="allAuthors_button">
       <property name="text">
        <string>All authors</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addGenre_button">
       <property name="text">
        <string>Add genre</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="allGenres_button">
       <property name="text">
        <string>All genres</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="statistics_button">
       <property name="text">
        <string>Statistics</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="settings_button">
       <property name="text">
        <string>Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="page"/>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
